designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DBFetchingProcessRepository.scala

```diff
@@ -136,6 +156,20 @@ abstract class DBFetchingProcessRepository[F[_]: Monad](
       }).map(_.toList)
   }
 
+  private def fetchLatestProcessByQueryAction[PS: ScenarioShapeFetchStrategy](
+      query: ProcessEntityFactory#ProcessEntity => Rep[Boolean],
+  )(
+      implicit loggedUser: LoggedUser,
+      ec: ExecutionContext
+  ): DBIOAction[List[PS], NoStream, Effect.All with Effect.Read] = {
+    (for {
+      latestProcesses <- fetchLatestProcessesQuery(query).result
+    } yield latestProcesses
+      .map { case ((_, _), processVersion) =>
```

Comment:
Let's move it inside for-comprehension

Reply:
done

Also,
- lazy val's instead of `def all` and `def byName` in the `ClassDefinitionSet`
- I don't know why previous pipeline failed. Some of the jobs failed because of failing containers, I suspect some technical problems on pipeline/runner, not issues with the code.